.github/workflows/ci_build.yml

```diff
@@ -0,0 +1,67 @@
+name: Build and Push
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+
+  build_push:
+    runs-on: ubuntu-latest
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
+    env:
+      ANSIBLE_CHATBOT_VERSION: "0.0.0"
```

Comment:
Would it be better to use `latest`?

Reply:
it doesn't mind, this image is not pushed anywhere....